python/pyiceberg/catalog/hive.py

```diff
@@ -233,24 +299,47 @@ def create_table(
             ValueError: If the identifier is invalid
         """
         database_name, table_name = self.identifier_to_database_and_table(identifier)
-        current_time_millis = int(time.time())
+        seconds_since_epoch = int(time.time() * 1000)
```

Comment:
This is in milliseconds, not seconds. I think you probably want to rename back to `current_time_millis`.

Reply:
Ugh, what a mess, thanks!